packages/synchro-charts/src/components/charts/sc-webgl-base-chart/sc-webgl-axis.spec.tsx

```diff
@@ -126,11 +126,11 @@ describe('initial render', () => {
 
   it('has expected y axis value labels', async () => {
     const { axis } = await axisSpecPage();
-    expect(axis.innerHTML).toContain('3,000');
-    expect(axis.innerHTML).not.toContain('3,025');
-    expect(axis.innerHTML).toContain('3,050');
+    expect(axis.innerHTML).toContain('3.00k');
+    expect(axis.innerHTML).not.toContain('3.0k');
```

Comment:
does 3.025 get rounded down?

Reply:
yeah, this is an example from 2025 to 3025 and they both round down 
<img width="546" alt="image" src="https://user-images.githubusercontent.com/28601414/199254171-0094a669-8df8-4467-ba94-6e4fdcf86a48.png">